custom-auth-lambda-triggers.tf

```diff
@@ -0,0 +1,35 @@
+resource "github_repository" "custom-auth-lambda-triggers" {
+  name        = "custom-auth-lambda-triggers"
```

Comment:
this name doesn't seem correct. Are these lambdas specific to the analytical environment?

Reply:
They are indeed - I will change it to be more specific